src/common/options/global.yaml.in

```diff
@@ -3682,6 +3682,29 @@ options:
   min: 896_M
   flags:
   - runtime
+- name: osd_memory_thread_cache_bytes
+  type: size
+  level: basic
+  desc: When tcmalloc is enabled it specifies thread cache size
+  long_desc: Sets tcmalloc.max_total_thread_cache_bytes value.
+    If set this will override settings privided via environment variable
```

Comment:
Some people might want to be able to override this via an environment variable, but I don't know if it actually matters.  @jdurgin @neha-ojha , do you have opinions?

Reply:
TCmalloc on library initialization reads environment variable and applies setting.
Then, in main() we read conf and apply setting again (if set).

Can you provide me with hint what "environment variable overrides ceph.conf" means in this context?

Reply:
@markhpc The description is very clear about which one will take precedence and there is also a way to continue to use the environment variable, if the user prefers to do so. Adding a note in the user facing release notes would also be good but I don't see any other issues.